### PR TITLE
add config file for iOS simulators

### PR DESCRIPTION
**Pull request by valentyn88**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,15 @@ saucectl run --test-env sauce
 
 ![sauce cloud example](assets/sauce_cloud_example.gif)
 
+### In Sauce Cloud with iOS Simulators
+# TODO (change gif file to ios + sauce mode)
```
Comment (alexplischke) [inline README.md:43]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -39,6 +39,15 @@ saucectl run --test-env sauce
...
+
+```shell
+saucectl run -c .sauce/config_ios.yml --test-env sauce
```
Comment (alexplischke) [inline README.md:46]: This `--test-env` is now deprecated, it'd be better to update the README and config accordingly 💡

File: README.md (hunk 1 above)
Comment (valentyn88) [reply README.md:43]: It's still draft. I left this comment not to forget to create a new gif file.

File: README.md (hunk 2 above)
Comment (valentyn88) [reply README.md]: @tianfeng92 has already created a PR to remove usage of `--test-end. We need to merge her changes firstly and after that, I can update my PR.

**Comment by valentyn88** [comment]
For iPhone 12 Simulator `#submit-button` is not visible and the test from `tests/example.test.js` constantly fails. I decided to add a new test file `tests/example.ios.test.js` that has a proper test for `iPhone 12 Simulator` and `iPad mini (5th generation) Simulator`.

File: .sauce/config.yml [hunk 3]
```diff
@@ -23,18 +23,18 @@ suites:
   - name: "Chrome using global mode setting" # Since the suite doesn't specify the `mode`, it'll inherit the mode specified via `defaults.mode` (see line number 3 and 4 of this config file).
     browserName: "chrome"
     src:
-      - "tests/*.test.js" # test files glob
+      - "tests/example.test.js" # test files glob
```
Comment (alexplischke) [inline .sauce/config.yml:26]: Hm. Why? 🤔 
What's so special about `example.test.js` vs. `example.ios.test.js` where one of them can only run on ios and the other one not?

I also don't think we quite a need a separate config for this. It's not like it's an entirely different setup. It's suggesting to the user that they need a separate config file entirely to be able to run their testcafe tests against iOS devices, but that shouldn't be the case.
I'd recommend to just add a suite to the existing config that utilizes ios 💪

Comment (valentyn88) [reply .sauce/config.yml]: As we agreed I added more tests and put the iOS suite to a config.yml

**Review by alexplischke — APPROVED** [review]

**Pull request merged**